docs/pages/machine-id/guides/jenkins.mdx

```diff
@@ -54,6 +68,10 @@ scope for server access, reduce the blast radius if one pipeline is
 compromised, and allow you to remotely audit and lock pipelines if you detect
 malicious behavior.
 
+<Figure align="left" bordered caption="Jenkins Deployments">
+  ![Jenkins Deployments](../../../img/machine-id/jenkins.png)
+</Figure>
```

Comment:
Should this be a separate commit? This way you can just revert the commit with the admonitions when we release 9.1.

Reply:
Good point, fixed.

You might already know this, but when you want to split changes like this into separate commits, you don't need to copy them over to a temporary file then copy them back. You can do a `get reset HEAD~1` then added the parts of `jenksin.mdx` sepearly  `git add -p`.